.evergreen/config.yml

```
@@ -134,15 +134,22 @@ functions:
         script: |
           ${PREPARE_SHELL}
 
-          # Install any go tools that we need. Use "go install" instead of "go get" to prevent
-          # modifying the go.mod file.
-          go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
-          go install github.com/walle/lll/...@latest
-
           # initialize submodules
           git submodule init
           git submodule update
 
+  install-linters:
+    - command: shell.exec
+      params:
+        working_dir: src/go.mongodb.org/mongo-driver
+        script: |
+          ${PREPARE_SHELL}
+
+          # Install linters. Use "go install" instead of "go get" to prevent modifying the go.mod
+          file.
```

```suggestion
          # file.
```
I think this is why the task is setup-failing.

---

Yep, I saw that too 😆 . Fixed!